tests/common.sh

```diff
@@ -76,8 +76,8 @@ function get_pod () {
   pod_name="False"
   until [ $pod_name != "False" ] ; do
     sleep $sleep_time
-    pod_name=$(kubectl get pods -l $1 --namespace ${3:-my-ripsaw} -o name | cut -d/ -f2)
-    if [ -z $pod_name ]; then
+    pod_name="$(kubectl get pods -l $1 --namespace ${3:-my-ripsaw} -o name | grep -vi Terminating | cut -d/ -f2)"
```

Comment:
This expression won't work as expected since the pod status is not shown when passing the flag `-o name` to kubectl.

Suggestion: using go-templates might be more robust with something like:
```
kubectl get pods -l $1 --namespace ${3:-my-ripsaw} -o go-template --template='{{range .items }}{{if eq .status.phase "Running"}}{{ .metadata.name }}{{" "}}{{end}}{{end}}'
```